src/Compilers/Test/Utilities/CSharp/UsesIsNullableVisitor.cs

```diff
@@ -26,15 +26,19 @@ internal static void GetUses(ArrayBuilder<Symbol> builder, Symbol symbol)
             visitor.Visit(symbol);
         }
 
+        private void Add(Symbol symbol)
```

Comment:
>private [](start = 8, length = 7)

Consider moving back to other `private` members.

Reply:
The reason I moved it up is that I needed to set a breakpoint in there multiple times, so having it at the top is convenient.

---
In reply to: [215758070](https://github.com/dotnet/roslyn/pull/29636#discussion_r215758070) [](ancestors = 215758070)